src/portalbackend/PortalBackend.Migrations/Seeder/Data/company_service_accounts.json

```
@@ -124,5 +124,45 @@
     "company_service_account_kind_id": 1,
     "client_id": "beb01d13-04e2-4a2b-a909-8b4166b3dcf7",
     "client_client_id": "sa-cl2-05"
+  },
+  {
+    "id": "48fb21ad-4439-455b-92bb-8c54801e3eab",
+    "name": "sa-cl7-cx-1",
+    "description": "Client for the BPDM Portal Gate to consume golden record data from the Pool.",
```

```suggestion
    "description": "Technical User for the BPDM Portal Gate to consume golden record data from the Pool.",
```

I'll make sure to also update the description for all new service accounts in portal-iam, didn't notice it during the review of https://github.com/eclipse-tractusx/portal-iam/pull/155